src/hooks/useKnockoutExplorer.ts

```diff
@@ -275,6 +277,10 @@ async function configurePortal(explorerParams: ExplorerParams): Promise<Explorer
   });
 }
 
+function shouldForwardMessage(message: PortalMessage) {
```

Comment:
Let's confirm the parent frame isn't sending anything down with this signature else we'll end up in an infinite loop. :)

Reply:
Good callout. I'll add protection around that anyway.